terraform/variables.tf

```diff
@@ -260,6 +266,12 @@ variable "bigquery_project_id" {
   type        = string
 }
 
+variable "bigquery_events_partition_granularity" {
+  description = "How granular you want partition to be. Ideally partitions will be 1-10 GB. Our org had ~750 MB/month. Valid values are DAY, HOUR, MONTH, and YEAR."
+  type        = string
+  default     = "MONTH"
```

Comment:
Can you add validation here?

Something like:

```
  validation {
    condition     = contains(["DAY", "HOUR", "MONTH", "YEAR"], var.bigquery_events_partition_granularity)
    error_message = "ERROR: bigquery_events_partition_granularity must be one of DAY, HOUR, MONTH, and YEAR"
  }

```

Reply:
Thanks for the suggestion, done